openapi.yaml

```diff
@@ -13021,6 +13021,22 @@ components:
         storageclass.fast_ssd:
           type: string
           description: storage class name to use to provision pvc
+        qovery.static_ip_mode:
+          type: boolean
+          x-stoplight:
+            id: i7u9vjd0o89g8
+          description: To limit public access from the internet to your Kubernetes cluster endpoint.
```

Comment:
can you add that whitelisted CIDR is defined in `k8s.api.allowed_public_access_cidrs` ?

Reply:
done